themes/docsy/layouts/partials/head.html

```diff
@@ -1,5 +1,6 @@
 <meta charset="utf-8">
 <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+<meta property="og:image" content="https://skywalking.apache.org/images/feather_400x400.png">
```

Comment:
Good PR, but please change the icon to SkyWalking's logo.

![pb-skywalking](https://user-images.githubusercontent.com/5441976/104861698-e26b7d00-596b-11eb-8159-81700a5a5cd0.png)